cob_bringup/drivers/sick_lms1xx.launch

```diff
@@ -6,11 +6,11 @@
 	<arg name="name"/>
 
 	<!-- load parameter -->
-	<rosparam ns="$(arg name)" command="load" file="$(arg pkg_hardware_config)/$(arg robot)/config/$(arg name).yaml"/>
+	<rosparam ns="$(arg name)/driver" command="load" file="$(arg pkg_hardware_config)/$(arg robot)/config/$(arg name).yaml"/>
```

Comment:
why did you add `driver` to the namespace?
this would be different from topic names, the other robots are using...